server/models/Request.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
```

Comment:
This file doesn't belong to this PR so I imagine this file was included for testing the endpoints. In this case, it would be better to keep a copy of this locally but exclude it from the PR/commits (i.e keep it untracked). You should be able to use `git rm` on your branch to untrack this file.

Reply:
I got confused. I have untracked this file.